src/pyOpenMS/tests/unittests/test_core_structures.py

```
@@ -0,0 +1,768 @@
+    ins = pyopenms.GaussFitResult(0.0, 0.0, 0.0)
+    ins.A = 5.0
+    ins.x0 = 5.0
+    ins.sigma = 5.0
+
+@report
+def testGaussFitter():
+    """
+    @tests: GaussFitter
+     GaussFitter.__init__
+    """
+    ins = pyopenms.GaussFitter()
```

Ruff (0.14.10)

Local variable `ins` is assigned to but never used

Remove assignment to unused variable `ins`

(F841)